CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.2.0] - 2021-06-30
+### Added
+- Adding TokenListController to fetch the token list from token services API ([#478](git+https://github.com/MetaMask/controllers/pull/478))
+- Feature/use @ethereumjs/tx ([#489](git+https://github.com/MetaMask/controllers/pull/489))
```

Comment:
This is not very human-readable :thinking: What impact does this have on users of the package?

Reply:
added those details in brackets

Reply:
This still isn't very human-readable. How would you expect people to interpret "Feature/use @ethereumjs/tx"? Why the "Feature/" prefix that we don't use anywhere else? What is `@ethereumjs/tx`?

Note that there's no reason to leave this matching the commit message. The purpose of this entry is to explain what changed - that's it.